left_to_do.txt

```diff
@@ -0,0 +1,3 @@
+- Discuss if we want to exclude judges who have not scored all teams from the scoring
```

Comment:
Yep, I think we should (if time permits) only calculate Judges scores for those Judges that have supplied scores to each team in the particular Hackathon, not any that are left-out.

Reply:
Cool. Will add a follow-up issue and PR for this.